global.css

```diff
@@ -8,7 +8,7 @@ button {
   text-overflow: ellipsis;
   padding: 2px 7px;
   border: 1px solid hsl(0, 0%, 62%);
-  font: 400 13.3333px Arial;
+  font: 400 calc(40 / 3)px Arial;
```

Comment:
Without a proof, I doubt there is any difference. And even if there is, we can simply use 13.3333333333333px which is the same as 40/3 due to 16-digit precision limit of IEEE754 float point numbers used internally by browsers.